src/main/avro/LandingZone.avsc

```diff
@@ -19,6 +19,32 @@
       "name": "kms_key_id",
       "type": "string"
     },
+    {
+      "name": "metadata",
+      "type": {
+        "name": "LandingZoneMetadataModel",
+        "type": "record",
+        "fields":
+        [
+          {
+            "name": "account_id",
+            "type": "string"
+          },
+          {
+            "name": "region",
+            "type": "string"
+          },
+          {
+            "name": "tags",
+            "type":
+            {
+              "type": "map",
```

Comment:
Yes, if we intend to use the version in the WSM (for instance, to store in the Cloud Context) we should make it a top-level metadata field.

My interpretation is that tags are meant to be opaque to the consumer and should not be used by the caller for anything other than tagging resources in a passthrough manner.   This also means that there will probably be some redundancy between data in the `Metadata` proper and the tag map ... this is OK.

@jwtracy does this align with your thinking?  Thoughts on adding (full | major.minor | major) version as a metadata field?

Reply:
Just added `major_version` as a field to `metadata` I don' t think we need to put the whole version string anywhere in there since the SoT is in the Terragrunt pipeline and the tags and metadata is more for consumer(lib)-facing bookkeeping.